packages/core/src/tracing/instrumentation/messaging/rdkafka.js

```diff
@@ -359,3 +364,10 @@ function findInstanaHeaderValues(instanaHeadersAsObject) {
 
   return { level, traceId, longTraceId, parentSpanId };
 }
+
+function logDeprecationKafkaAvroMessage() {
```

Comment:
Suggestion for function name `logKafkaAvroDeprecationWarning`

Reply:
haha yeah you are right, but I am too lazy to change it now :D